cluster/cluster_interface.ml

```diff
@@ -36,13 +36,18 @@ type node = {addr: address; id: nodeid} [@@deriving rpcty]
 
 type all_members = node list [@@deriving rpcty]
 
+type pems = {cn: string; blobs: string list} [@@deriving rpcty]
```

Comment:
PEM is just a format; would it be more appropriate to calls this `certs` and the blobs `pems`?

Reply:
Each blob is supposed to be a pem file string (so a blob contains both a cert and a private key)